src/lib.rs

```diff
@@ -473,6 +473,7 @@ extern crate tokio;
 extern crate tokio_fs;
 extern crate tokio_io;
 extern crate tower_service;
+extern crate url;
```

Comment:
Any reason to include url when all we are using is the re-exported [percent-encoding](https://docs.rs/percent-encoding) crate?

Reply:
Fixed, thanks